md/release-notes.md

```diff
@@ -214,14 +214,31 @@ Debug, the option to run a diagram without its connectors, is not supported anym
 * BS-19105	Studio Import: unable to recognize git repositories after studio migration
 * BS-19158	Default SAML configuration doesn't work with AD FS
 
+### Fixes in Bonita 7.8.3 (2019-03-07)
+#### Fixes in Studio component
+BST-49	Allow condition on a sequence flow that connect a BPMN element to a parallel gateway	
+BST-77	Diagram overview broken after pool sizing in the Studio	
+BST-94	Bonita la builder doesn't work if a process contains a decision table	
+BST-96 Impossible to import a BOS renamed in 7.8.0 in other Community Studios
+BST-100 Project explorer makes Studio considerably slower on Windows	
+
+#### Fixes in UI Designer component
+UID-8	Display placeholder option on Datepicker widget if an expression is set	
+UID-11	Bonita API resource Template path truncated when exported	in page.properties
```

Comment:
```suggestion
UID-11	Bonita API resource Template path truncated when exported in `page.properties`
```

Reply:
hard to see it in the md file. I've tried to erase all spaces to rewrite it.

Reply:
done